### PR TITLE
Sanity fix

### PR DESCRIPTION
**Pull request by GunaKKIBM**

### CODE REVIEW

File: pkg/mountmanager/fake-safe-mounter.go [hunk 1]
```diff
@@ -39,8 +39,8 @@ func NewFakeNodeMounter() Mounter {
 // NewFakeSafeMounter ...
 func NewFakeSafeMounter() *mount.SafeFormatAndMount {
 	fakeMounter := &mount.FakeMounter{MountPoints: []mount.MountPoint{{
-		Device: "valid-devicePath",
```
Comment (sameshai) [inline pkg/mountmanager/fake-safe-mounter.go:42]: @GunaKKIBM I think we use same fakes for block ? Will it impact can you check ?

File: pkg/mountmanager/fake-safe-mounter.go [hunk 2]
```diff
@@ -39,8 +39,8 @@ func NewFakeNodeMounter() Mounter {
...
-		Path:   "valid-vol-path",
+		Device: "some/target",
```
Comment (sameshai) [inline pkg/mountmanager/fake-safe-mounter.go:42]: Same Here

**Review by sameshai — CHANGES_REQUESTED** [review]
Please check the comments

File: pkg/mountmanager/fake-safe-mounter.go (hunk 2 above)
Comment (GunaKKIBM) [reply pkg/mountmanager/fake-safe-mounter.go:42]: This test was failing for Block too. Will raise another PR for sanity, which will work well for both file and block driver.

**Review by sameshai — APPROVED** [review]

**Review by ambiknai — APPROVED** [review]

**Pull request merged**